RevenueCatUI/Templates/V2/Components/Packages/Package/PackageComponentView.swift

```diff
@@ -12,7 +12,7 @@
 //  Created by Josh Holtz on 9/27/24.
 
 import Foundation
-import RevenueCat
+@_spi(Internal) import RevenueCat
```

Comment:
This `@_spi(Internal)` would not be needed, unless we make `PaywallComponent.PackageComponent`'s `applePromoOfferProductCode` `@_spi(Internal)`